patches/tModLoader/Terraria/Localization/Content/ru-RU/tModLoader.json

```diff
@@ -418,37 +418,37 @@
+		"DefaultExtraMessage": "восстановите установку Terraria, затем установите tModLoader в Steam или следейте инструкциям из README.txt для ручной установки.",
+		"ClientLogHint": "{0}\n\nФайл client.log с информацией об ошибке создан в\n{1}\n(Прикрепите этот файл, если попросите о помощи)",
 		"OutOfMemoryHint": "\n\nПопробуйте использовать меньше модов либо определите какой мод/моды потребляет много памяти. Опционально, установите 64-битный tModLoader",
 		"ModExceptionHint": "\n\nЭто скорее всего проблема мода(ов). Отключите моды по одному и проверьте, осталась ли проблема",
 		"OneDriveHint": "\n\nПопробуйте установить/включить OneDrive. Нажмите правую кнопку мыши по папке \"Документы\" и включите \"Всегда сохранять на этом устройстве\"",
 		"AntivirusHint": "\n\nЕсли у Вас есть антивирус, добавьте Terraria в его список исключений/белый список",
-		// "TypeInitializationHint": "\n\nTip: Restart the game. If this issue persists, try uninstalling Terraria completely, then reinstalling through Steam, and then reinstalling tModLoader",
-		// "TagsContent": "New Content",
-		// "TagsContentDescription": "Adds new content (i.e. items, weapons, npcs, etc.) to the game",
-		// "TagsUtility": "Utilities",
-		// "TagsUtilityDescription": "Adds utility-oriented content or features",
-		// "TagsLibrary": "Library",
-		// "TagsLibraryDescription": "Provides additional API & features useful to modders",
-		// "TagsQoL": "Quality Of Life",
-		// "TagsQoLDescription": "Features or changes that reduce time spent on typically less fun tasks.",
-		// "TagsGameplay": "Gameplay Tweaks",
-		// "TagsGameplayDescription": "Tweaks that change or improve vanilla gameplay",
-		// "TagsAudio": "Audio Tweaks",
-		// "TagsAudioDescription": "Tweaks that change existing audio (i.e. new boss themes, etc.)",
-		// "TagsVisual": "Visual Tweaks",
-		// "TagsVisualDescription": "Tweaks that change existing visualization (i.e. particle effects, etc.)",
-		// "TagsLang": "Localization Support",
-		// "TagsLangDescription": "This mod supports multiple languages",
-		// "TagsGen": "Custom World Gen",
-		// "TagsGenDescription": "Changes the way the world is generated (i.e. ores, new structures, etc.)"
+	 "TypeInitializationHint": "\n\nПерезапустите игру. Если проблема останется, попробуйте полностью удалить Terraria, затем переустановить через Steam, и затем переустановить tModLoader",
+		"TagsContent": "Новый контент",
+		"TagsContentDescription": "Добавляет новый контент (т.е. предметы, оружия, НИПов, и т.д.) в игру",
+		"TagsUtility": "Утилиты",
+		"TagsUtilityDescription": "Добавляет утилитарный контент или функции",
+		"TagsLibrary": "Библиотеки",
+		"TagsLibraryDescription": "Предоставляет дополнительные API и функции, полезно для разработчиков",
+		"TagsQoL": "Quality Of Life",
```

Comment:
I noticed that you forgot to translate the line "TagsQoL": "Quality Of Life".

Reply:
Idk how to properly translate id and QoL in English is understandable too

Reply:
> Idk how to properly translate it and QoL in English is understandable too

Oh okay